src/main.ts

```diff
@@ -27,7 +27,7 @@ async function run() {
         if (invalidJsons.length > 0) {
             core.setFailed('Failed to validate all JSON files.');
         } else {
-            core.info(`✅ All files were validate succesfully.`);
+            core.info(`✅ All files where validated succesfully.`);
```

Comment:
Should be `All files were validated successfully`

Reply:
You are maybe right but please post links so I can improve my knowledge

Reply:
https://www.dictionary.com/browse/where?s=t
https://www.dictionary.com/browse/were?s=t

Reply:
Thank you, fixed